### PR TITLE
update user guide

### PR DESCRIPTION
**Pull request by shane-huang**
changed pages are:
1. add `IPEX-LLM` in `Local Model Inference Runtimes Supported` in https://docs.dify.ai/getting-started/readme/specifications-and-technical-features#technical-features
2. add a row for `IPEX-LLM` in https://docs.dify.ai/getting-started/readme/model-providers
3. add a new page in model-configurations https://docs.dify.ai/tutorials/model-configuration
4.add `IPEX-LLM` in model-configuration index page https://docs.dify.ai/tutorials/model-configuration

### CODE REVIEW

File: en/tutorials/model-configuration/ipex-llm.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Integrating Local Models Deployed with IPEX-LLM
+
+[IPEX-LLM](https://github.com/intel-analytics/ipex-llm) is a PyTorch library for running LLM on Intel CPU and GPU (e.g., local PC with iGPU, discrete GPU such as Arc, Flex and Max) with very low latency.
```
Comment (Oscilloscope98) [inline en/tutorials/model-configuration/ipex-llm.md:3]: Arc -> Arc A-Series ?

As MTL iGPU is also named as Intel Arc Graphics

File: en/tutorials/model-configuration/ipex-llm.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+## Quick Integration
+
+### Install and Run Ollama with IPEX-LLM
+
+Follow the instructions in [this guide](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/Quickstart/ollama_quickstart.html) to start an Ollama service to run local LLMs on Intel GPU. 
+
+### Integrate Ollama with Dify
+
+Next, integrate with Dify following the same steps as in [Ollama integration guide](https://docs.dify.ai/tutorials/model-configuration/ollama).
```
Comment (Oscilloscope98) [inline en/tutorials/model-configuration/ipex-llm.md:15]: Should we have a relative link here? If the ollama integration guide belongs to the same document with our newly added one :)

File: en/tutorials/model-configuration/ipex-llm.md (hunk 1 above)
Comment (shane-huang) [reply en/tutorials/model-configuration/ipex-llm.md:3]: I used the same sentence as in IPEX-LLM README

File: en/tutorials/model-configuration/ipex-llm.md (hunk 2 above)
Comment (shane-huang) [reply en/tutorials/model-configuration/ipex-llm.md:15]: dify didn't provide instructions about how to build its docs, and its contribution guide does not mention about docs either. 

I checked other doc pages to find out how they link to their own document - they used absolute link. So I used absolute too.

**Pull request merged**